test-files/cmd/slow/ServiceOsmApiDbHootApiDbPoiPolygonConflateTest/output.sql

```diff
@@ -1,2463 +0,0 @@
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=202;
-/* delete node 203*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (203, 377641954, -1224201365, 3, false, (now() at time zone 'utc'), 1301409561, 2);
-DELETE FROM current_node_tags WHERE node_id = 203;
-DELETE FROM node_tags WHERE node_id = 203;
-DELETE FROM current_way_nodes WHERE node_id=203;
-DELETE FROM current_relation_members WHERE member_type = 'Node' AND member_id = 203;
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=203;
-/* delete node 204*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (204, 377626708, -1224199035, 3, false, (now() at time zone 'utc'), 1301409560, 2);
-DELETE FROM current_node_tags WHERE node_id = 204;
-DELETE FROM node_tags WHERE node_id = 204;
-DELETE FROM current_way_nodes WHERE node_id=204;
-DELETE FROM current_relation_members WHERE member_type = 'Node' AND member_id = 204;
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=204;
-/* delete node 205*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (205, 377642041, -1224200053, 3, false, (now() at time zone 'utc'), 1301409561, 2);
-DELETE FROM current_node_tags WHERE node_id = 205;
-DELETE FROM node_tags WHERE node_id = 205;
-DELETE FROM current_way_nodes WHERE node_id=205;
-DELETE FROM current_relation_members WHERE member_type = 'Node' AND member_id = 205;
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=205;
-/* delete node 206*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (206, 377626613, -1224199062, 3, false, (now() at time zone 'utc'), 1301409560, 2);
-DELETE FROM current_node_tags WHERE node_id = 206;
-DELETE FROM node_tags WHERE node_id = 206;
-DELETE FROM current_way_nodes WHERE node_id=206;
-DELETE FROM current_relation_members WHERE member_type = 'Node' AND member_id = 206;
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=206;
-/* delete node 207*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (207, 377642219, -1224197369, 3, false, (now() at time zone 'utc'), 1301409561, 2);
-DELETE FROM current_node_tags WHERE node_id = 207;
-DELETE FROM node_tags WHERE node_id = 207;
-DELETE FROM current_way_nodes WHERE node_id=207;
-DELETE FROM current_relation_members WHERE member_type = 'Node' AND member_id = 207;
-UPDATE current_nodes SET changeset_id=3, visible=false, version=2 WHERE id=207;
-/* delete node 208*/
-INSERT INTO nodes (node_id, latitude, longitude, changeset_id, visible, "timestamp", tile, version) VALUES (208, 377626692, -1224199156, 3, false, (now() at time zone 'utc'), 1301409560, 2);
```

Comment:
Actually, its ok.  The outputs of the test are supposed to be .osc.sql and .osc.  I must have made an extra unused .sql file at some point on the develop branch that was a copy of the .osc.sql file and am deleting it here.